rules/AWS_S3_Bucket_Open_ACL.sql

```diff
@@ -0,0 +1,57 @@
+CREATE OR REPLACE VIEW rules.aws_s3_bucket_open_acl_violation_query COPY GRANTS
+  COMMENT='S3 Bucket ACLs can grant or deny access to specific entities. This rule checks for public bucket access.
+  @id 6rn7ad2y7tv
```

Comment:
What is this id syntax? What is it for?

Reply:
The id in the comment is for the UI to use. Our BI also references it. Having the identifier in their let's us avoid pulling from the database for that info. Value should be the same as the query_id.

Reply:
is this just a random value?

Reply:
Yup, exactly. Clicking on the +Query button in the Alerts or Violations section will generate boilerplate rule with a unique value in there.

Reply:
It's required for BI because of a permissions limitation on our end -- view-only roles can read the comment of a view but not the SQL.